src/front/wgsl/mod.rs

```diff
@@ -3470,33 +3470,41 @@ impl Parser {
                         lexer.expect(Token::Paren(')'))?;
 
                         let accept = self.parse_block(lexer, context.reborrow(), false)?;
+
                         let mut elsif_stack = Vec::new();
                         let mut elseif_span_start = lexer.current_byte_offset();
-                        while lexer.skip(Token::Word("elseif")) {
-                            let mut sub_emitter = super::Emitter::default();
+                        let mut reject = loop {
+                            if lexer.skip(Token::Word("else")) {
```

Comment:
can we rewrite this with early exists instead?
```rust
if !lexer.skip(...) {
  break ...;
}
```

Reply:
Good point! Pushing the changes...